tap_google_ads/streams.py

```diff
@@ -43,6 +45,18 @@ def get_conversion_window(config):
 
     raise RuntimeError("Conversion Window must be between 1 - 30 inclusive, 60, or 90")
 
+
+def get_request_timeout(config):
```

Comment:
If we gave `make_request` a default config param of `{}`, we'd be able to fall back to the default without giving an empty config. That would mean you wouldn't have to change the function signature, and every existing call to this function would remain unchanged. Might be a bit nit-picky, so I'll leave it up to you if you want to go back and change all the function calls back.